src/Helper/ExportHelper.php

```diff
@@ -45,25 +48,16 @@ class ExportHelper extends AbstractExportHelper
      * @inheritdoc
      * @suppress PhanAccessMethodInternal, PhanUndeclaredClassConstant, PhanUndeclaredClassMethod
      */
-    public function encrypt($secret, $data)
+    public function encrypt(string $secret, $data)
     {
-        //Check if phpseclib v3 is used
-        //needed for comaptibility with Magento 2.4 versions
-        if (class_exists("phpseclib3\Crypt\AES")) {
-            $iv = \phpseclib3\Crypt\Random::string(16);
-            $cipher = new \phpseclib3\Crypt\AES('cbc');
-        } else {
-            $iv = \phpseclib\Crypt\Random::string(16);
-            $cipher = new \phpseclib\Crypt\AES(\phpseclib\Crypt\Base::MODE_CBC);
-        }
-
+        $cipher = new AES('gcm');
+        $cipher->setKeyLength(256);
         $cipher->setKey($secret);
-        $cipher->setIV($iv);
+        $iv = Random::string(12);
```

Comment:
minor: from the functionality perspective, looks good. I'd use constants for `nonce` and `tag` lengths